apps/accounts/views/upstream_login.py

```diff
@@ -0,0 +1,20 @@
+from django.shortcuts import render
+from django.conf import settings
+import urllib.request as req
```

Comment:
On the Login Page: 

If we are using MyMedicare.gov login we should reduce the size of the original login box and re-title it "Developer Login".